.travis.yml

```diff
@@ -14,8 +14,6 @@ services:
 language: python
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27
```

Comment:
Is there a reason to keep this running on Py2?  I can revert it if there is

Reply:
@dralley So that is the Ansible management node (the Ubuntu host), not the managed system (the container.)

We need to assume that Python 2.7 will be used on the management node for a long time. I have not seen any real issues with 2.7 vs 3.x on it, but we should still keep it.